bin/index.js

```diff
@@ -1,15 +1,14 @@
 #! /usr/bin/env node
-const got = require('got')
 
-const { backupMessages } = require("./backupMessages");
+const { prefixCommand, defaultCommand } = require("./commands");
 
-const generateRandomNumber = Math.floor(Math.random() * (backupMessages.length - 1));
-
-(async () => {
-	try {
-		const response = await got('http://whatthecommit.com/index.txt');
-		console.log(response.body);
-	} catch (error) {
-		console.log(backupMessages[generateRandomNumber]);
-	}
-})();
\ No newline at end of file
+switch (process.argv[2]) {
+  case 'prefix':
```

Comment:
this could be a const

Reply:
[1f8de177812046f137a9660b06f3fd82d5892dc7](https://github.com/ellie-keen/lolcommit/pull/11/commits/1f8de177812046f137a9660b06f3fd82d5892dc7)